basics/concepts/segment-threshold.md

```diff
@@ -0,0 +1,26 @@
+---
+description: In this Apache Pinot concepts guide, we'll learn how segment threshold works.
+---
+
+# Segment Threshold
+
+The segment threshold determines when a segment is committed in real-time tables.
+
+When data is first ingested from a streaming provider like Kafka, it gets stored in a consuming segment. 
+The data in the consuming segment is stored on the disk of the server(s) that are processing a particular partition from the streaming provider.
+
+However, it's not until a segment is committed that the segment is written to the [deep store](https://docs.pinot.apache.org/basics/components/deep-store). 
```

Comment:
and then the segment is written to deep store?

Reply:
i may be missing something here--did we say something earlier that may make folks think that a segment would be written somewhere before it's committed to something?